ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/sts/custom-endpoints/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-provision-chain.yaml

```diff
@@ -0,0 +1,36 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-sts-custom-endpoints-provision
+  steps:
+    - ref: aws-provision-vpc-disconnected
+    - chain: aws-provision-bastionhost
+    - chain: mirror-images-by-oc-adm
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-manual-creds
+    - ref: ipi-conf-aws-oidc-creds-provision
+    - ref: ipi-conf-aws
+    - ref: ipi-conf-aws-custom-endpoints
+    - ref: ipi-conf-mirror
+    - ref: ipi-install-monitoringpvc
+    - ref: proxy-config-generate
+    - ref: ipi-install-install-aws
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource-disconnected
+    - ref: set-sample-operator-disconnected
+    - ref: ipi-install-times-collection
```

Comment:
I am trying to tidy up the workflows in https://github.com/openshift/release/pull/33289, suggest to move `ref: ipi-install-times-collection` behind `ref: ipi-install-install-aws` to keep consistent.